SConstruct

```diff
@@ -1212,6 +1212,11 @@ for option, default in python_options:
             print("WARNING: Ignoring option 'python3_{0}'"
                 " because 'python_{0}' was also set.".format(option))
 
+# Prevent setting prefix for Cantera installation to source directory
+if os.getcwd() == os.path.abspath(env['prefix']):
```

Comment:
I'm not sure that `os.getcwd()` is the right thing here, since it may be possible to run SCons from outside the source directory (I'm not sure I've never tried). If the problem is the `'prefix'` is the same as the source directory, it would be better to check whether the `'prefix'` was the same as the directory where `SConstruct` is located.

If this is an error, should the exit code be 1 instead of 0?

Reply:
True! Both are easily addressed.